packages/e2e-tests/specs/editor/blocks/heading.test.js

```diff
@@ -60,14 +58,15 @@ describe( 'Heading', () => {
 		await colorPanelToggle.click();
 
 		const [ customTextColorButton ] = await page.$x(
-			`${ TEXT_COLOR_UI_X_SELECTOR }${ CUSTOM_COLOR_BUTTON_X_SELECTOR }`
+			`${ CUSTOM_COLOR_BUTTON_X_SELECTOR }`
 		);
+
 		await customTextColorButton.click();
 		await page.click( COLOR_INPUT_FIELD_SELECTOR );
 		await pressKeyWithModifier( 'primary', 'A' );
-		await page.keyboard.type( '#181717' );
+		await page.keyboard.type( '#7700ff' );
 		await page.click( '[data-type="core/heading"] h3' );
-		await page.waitForSelector( '.component-color-indicator[aria-label="(Color: #181717)"]' );
+		await page.waitForSelector( '.component-color-indicator[aria-label="(Color: #7700ff)"]' );
```

Comment:
Any particular reason for changing the colour?

Reply:
I was watching the tests via the `PUPPETEER_HEADLESS=false PUPPETEER_SLOWMO=80 npm run test-e2e:watch` command, and changing it to a color like `#181717` didn't give much of a visual indication that the color had changed. However, changing it to `#7700ff`, a bright purple, made it obvious that the color change had applied successfully :)